framework/encode/vulkan_state_writer.cpp

```diff
@@ -1755,7 +1755,7 @@ void VulkanStateWriter::WriteImageMemoryState(const VulkanStateTable& state_tabl
             VkMemoryPropertyFlags memory_properties = 0;
             if (memory_wrapper != nullptr)
             {
-                GetMemoryProperties(device_wrapper, memory_wrapper, state_table);
+                memory_properties = GetMemoryProperties(device_wrapper, memory_wrapper, state_table);
```

Comment:
I caught this while reading the code. It is not directly related to the issue this patch intents to fix.

Reply:
Line was changed by 009b820f924dcb269365e71bebc9e100fb4358fb (@locke-lunarg )